src/utils/configuration.ts

```diff
@@ -432,6 +434,11 @@ const isPlatformAuthEnabled = (): boolean => {
   return getCloudConfigurationValue('platformAuth', false)
 }
 
+const getFaqUrl = (hash: string): string => {
```

Comment:
I don't love this. I'd suggest using a purpose-specific setting and leveraging a default URL that uses the destination redirection pattern (as used in the foldout links):

https://github.com/stateful/docs.runme.dev/blob/main/vercel.json#L3-L62

That way it's easy to change the final docs destination and move it around even for older packaged versions.